### ISSUE TITLE
code implementation error

### ISSUE THREAD

**Issue by PuNeal** [openppl-public/ppq#59]
Hi, I found an error in [https://github.com/openppl-public/ppq/blob/df53c934748cb0ded7bdd0089398f3053265dcc5/ppq/quantization/algorithm/equalization.py#L328](url)

It sholud be `torch.mean(torch.square(params), axis=aggerate_axis)`.

**Comment by ZhangZhiPku** [comment]
Fixed with this : #60
